python/caffe/_caffe.cpp

```diff
@@ -25,6 +25,10 @@
 #define PyArray_SetBaseObject(arr, x) (PyArray_BASE(arr) = (x))
 #endif
 
+// Hack to convert macro to string
+#define STRINGIZE(m) #m
+#define STRINGIZE2(m) STRINGIZE(m)
```

Comment:
The names are obscure here... how about calling the first macro `STRINGIFY` (which is the terminology from the cpp manual) and the second `AS_STRING` (which makes its functionality more clear)?